client/src/components/RecipeGrid.js

```diff
@@ -20,7 +20,7 @@ class RecipeGrid extends Component {
     }
     return recipes.map((recipe, index) => {
       return (
-        <Recipe downvoteRecipe={ this.props.downvoteRecipe } upvoteRecipe={ this.props.upvoteRecipe } recipe={ recipe}  key={ index } index={ index } />
+        <Recipe addFavoriteRecipe = {this.props.addFavoriteRecipe} downvoteRecipe={ this.props.downvoteRecipe } upvoteRecipe={ this.props.upvoteRecipe } recipe={ recipe}  key={ index } index={ index } />
```

Comment:
You can break each attributes/property down to a separate line to make the code cleaner.